service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceMethod.java

```diff
@@ -22,9 +23,9 @@
 {
     private IotHubConnectionString iotHubConnectionString = null;
     private Integer requestId = 0;
-    private static final int DEFAULT_RESPONSE_TIMEOUT = 30; // default response timeout is 30 seconds
-    private static final int DEFAULT_CONNECT_TIMEOUT = 0;
```

Comment:
These default values have changed to 24 seconds each to follow the RegistryManager pattern

Reply:
nit: If we are changing the default values, can we default to a round number instead? 😄